### DISCUSSION TITLE
Recipe in general and quantum armors speed

### DISCUSSION THREAD

**Discussion by lolersaurus** [TechReborn/TechReborn#3107] [Q&A]
Is there a way for a player with very minimum knowledge in modding to change recipes? I came from Thermal and Mekanism to Tech Reborn and found Tech Reborn's recipes are super harsh and tedious e.g. I find that it's not worth it to make item storage compared to default minecraft chest or shulker. Then comes the fusion reactor where you need lots of beryllium buckets(instead of the stackable tin tubes), clicking and crafting the parts one by one over and over. That said, I know nothing about gregtech, was gregtech also this harsh and tedious?

Secondly, is it possible to change the quantum armor speed into two speed mode? Ideally speaking, the walking speed of Beacon's Speed II while the running speed stay quantum armor's default speed. Currently, I struggle so much trying to enter a 1x1 hole without taking my quantum pants off.

**Comment by Thepigcat76** [comment]
For changing recipes you can use crafttweaker or datpacks. I wouldnt recommend using kubejs because it breaks many existing TR recipes

**Reply by lolersaurus** [reply]
Tried crafttweaker but it unfortunately doesn't seem to support 1.16.5 fabric. Is datpacks written as such? I couldn't find one.

**Comment by Ayutac** [comment]
>  where you need lots of beryllium buckets(instead of the stackable tin tubes)

you can automize this. besides that, you cannot change the recipe to cells because vanilla crafting grid does not respect nbt
